TODO.md

```diff
@@ -1 +1,2 @@
-See [issues](https://github.com/uniqush/uniqush-push/issues)
+Use NSQ to send push messages
+Use SQLite, MySQL or MSSQL as database backend
```

Comment:
For one, there's already an issue that I'm considering to use a relational database to store our data. But that is considered as a plan for version 2.0. The current database interface is not suitable for relational database so far as I can tell.